using_the_api.md

```diff
@@ -99,6 +100,33 @@ The structure of the post request allows for a flexible and complex query with b
 
 * `page` - If your request requires pagination, this parameter specifies the page of results to return. Default: 1
 * `limit` - The maximum length of a page in the response. Default: 100
+* `unique_values` - A list of fields for which you would like to know the unique values and how many items have that value. An example response with that value would be:
```

Comment:
Thanks for keeping the docs up to date! You hint at this towards the end of the unique values description, but do you think it's worth explicitly stating that `unique_value` tallies things after filters are applied? 

If someone tries to `exclude` a field and do a `unique_value` on that same field, is that allowable?